docs/source/recipes/lora_finetune_single_device.rst

```diff
@@ -5,11 +5,10 @@ LoRA Single Device Finetuning
 =============================
 
 This recipe supports finetuning on next-token prediction tasks using parameter efficient fine-tuning techniques (PEFT)
-such as `LoRA <https://arxiv.org/abs/2106.09685>`_ and `QLoRA <https://arxiv.org/abs/2305.14314>`_.  These techniques
+such as :ref:`glossary_lora` and :ref:`glossary_qlora`. These techniques
```

Comment:
nit: why not keep the links to the original papers here?

Reply:
because we have pages for them, explaining the details, how to use it, the these pages have hte link for the papers.